packages/code-du-travail-frontend/src/search/AddressResults.js

```diff
@@ -84,15 +90,22 @@ function AddressCard({ data }) {
   return (
     <Card>
       <CardTitle>{data.title}</CardTitle>
-      <CardAddress>
+      {data.subtitle && <CardSubtitle>{data.subtitle}</CardSubtitle>}
+      <CardAddress
```

Comment:
maybe ensure we have some lat/lon before rendering `CardAddress` ?

Reply:
@revolunet since we retrieve data from annuaire, I'm pretty confident that address exist
also if there is no coord, indexing will break